README.md

```diff
@@ -67,4 +67,76 @@ To validate all packages in the dist folder run `make validate`.
 
 ```shell
 make validate
-```
\ No newline at end of file
+```
+
+# QA / CICD
```

Comment:
Nit; Why does it have to be specifically QA? I think it's applicable to everyone who uses corral packages